source/frontend/src/hooks/providers/useProjectProvider.ts

```diff
@@ -30,8 +31,21 @@ export const useProjectProvider = () => {
       }, []),
     });
 
-  return {
-    retrieveProjectProducts,
-    retrieveProjectProductsLoading,
-  };
+  const addProjectApi = useApiRequestWrapper<
+    (...args: any[]) => Promise<AxiosResponse<Api_Project, any>>
```

Comment:
The parameter types are known so no need to use '(...args: any[])'

Reply:
similar approach used in AcquistionProvider...

Reply:
I think is an older pattern. Should be fixed there as well eventually.

Reply:
Still needs to be updated

Reply:
Updated